### DISCUSSION TITLE
how do you create you website?

### DISCUSSION THREAD

**Discussion by stefaneicher** [python-poetry/poetry#4157] [General]
Hi,
I started with poetry and for our project  https://github.com/disroop/mumoco we want to create a website and yous looks great!
how do you create the website https://python-poetry.org/docs from the md files?

Regards Stefan

**Comment by finswimmer** [comment]
Hey @stefaneicher,

I will convert this issue to a github discussion.

fin swimmer

**Comment by sdispater** [comment]
The website design is custom made.

Previously, the documentation was generated by `mkdocs` and packaged in the website built by the `jekyll` static site generator.

Recently, we changed the stack to the `hugo` static site generator which also builds the documentation.

**Reply by stefaneicher** [reply]
ok thanks for the answer. Regards Stefan
